File: PHP/uberVan.php
<?php
require_once('car.php');
class UberVan extends Car
{
    public $typeCarAccepted;
    public $seatsMaterial;

    public function __construct($license, $driver,$typeCarAccepted,$seatsMaterial)
    {
        parent::__construct($license, $driver); # es como super en JAVA
        $this->typeCarAccepted = $typeCarAccepted;
        $this->seatsMaterial = $seatsMaterial;
    }
}

File: Java/UberVan.java
package Java;

import java.util.ArrayList;
import java.util.Map;

public class UberVan extends Car {

    Map<String, Map<String,Integer>> typeCarAcccepted;
    ArrayList<String> seatsMaterial;

    public UberVan(String license, Account driver,Map<String, Map<String,Integer>> typeCarAcccepted,ArrayList<String> seatsMaterial){
        super(license, driver);//para llamar al construcctor de Car
        this.typeCarAcccepted = typeCarAcccepted;
        this.seatsMaterial = seatsMaterial;
    }
}
